otter/metrics.py

```diff
@@ -180,35 +191,40 @@ def get_all_metrics(cass_groups, authenticator, service_mapping, region,
     def req_func_for_tenant(tenant_id):
         return get_request_func(authenticator, tenant_id, metrics_log,
                                 service_mapping, region)
-    effs = get_all_metrics_effects(cass_groups, req_func_for_tenant, metrics_log, _print=_print)
+    effs = get_all_metrics_effects(
+        cass_groups, req_func_for_tenant, metrics_log, _print=_print)
     d = _perform_limited_effects(effs, clock, 10)
     d.addCallback(filter(lambda x: x is not None))
     return d.addCallback(lambda x: reduce(operator.add, x, []))
 
 
-def add_to_cloud_metrics(request_func, conf, region, total_desired, total_actual,
-                         total_pending, log=None):
+def add_to_cloud_metrics(request_func, conf, region, total_desired,
+                         total_actual, total_pending, log=None):
     """
-    Add total number of desired, actual and pending servers of a region to Cloud metrics
+    Add total number of desired, actual and pending servers of a region
+    to Cloud metrics
 
-    :param dict conf: Metrics configuration, will contain credentials used to authenticate
-                        to cloud metrics, and other conf like ttl
-    :param str identity_url: URL of identity API to authenticate users given in `conf`
+    :param dict conf: Metrics configuration, will contain tenant ID of tenant
+                      used to ingest metrics and other conf like ttl
     :param str region: which region's metric is collected
-    :param int total_desired: Total number of servers currently desired in a region
-    :param int total_actual: Total number of servers currently there in a region
-    :param int total_pending: Total number of servers currently building in a region
-    :param _treq: Optional treq implementation for testing
+    :param int total_desired: Total number of servers currently desired
+                              in the region
```

Comment:
flake8 doesn't by default have a docstring linter. pep257 takes care of part of it, but not all of it. sphinx has some plugins, but they're not installed yet.

After those few PRs, we're definitely not done with a fully-functional linter.